templates/server/server.tmpl

```diff
@@ -0,0 +1,79 @@
+/*
+* This file autogenerated;
+*
+* DO NOT EDIT
+*
+* {{ .Info.Title }}
+* Version: {{ .Info.Version }}
+*
+*/
+
+package server
+
+{{ $cName := (printf "HTTP%sServer" (goName .Info.Title false) ) }}
```

Comment:
honestly, i cant see at this moment why we need receiver for handlers helpers. those helpers are pretty much stateless and works only with request, you know..